### PR TITLE
[Docs] Update working cases example outputs for number_to_* helpers

### PR DESCRIPTION
**Pull request by JuanitoFatas**
This gives them a better idea of what the helper will do when reading the docs.

### Motivation / Background

[I was browsing the docs](https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_with_delimiter), and it would be more useful if the working example docs gave real example output.

### Detail

This Pull Request changes example output of `number_to_*` helpers.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [ ] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [ ] Tests are added or updated if you fix a bug or add a feature.
* [ ] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: actionview/lib/action_view/helpers/number_helper.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def initialize(number)
       # Additionally, supports a +:raise+ option that will cause
       # InvalidNumberError to be raised if +number+ is not a valid number:
       #
-      #   number_to_phone("12x34")              # => "12x34"
+      #   number_to_phone("1234567890")         # => "123-456-7890"
```
Comment (p8) [inline actionview/lib/action_view/helpers/number_helper.rb:32]: These examples show that passing a invalid number just returns the input by default.
So I think the examples are correct.

We can add a valid example after the opening line.
```markdown
# Delegates to ActiveSupport::NumberHelper#number_to_phone.
#
#   number_to_phone("1234567890") # => "123-456-7890"
#
```

**Review by p8 — CHANGES_REQUESTED** [review]

File: actionview/lib/action_view/helpers/number_helper.rb [hunk 2]
```diff
@@ -26,10 +26,11 @@ def initialize(number)
 
       # Delegates to ActiveSupport::NumberHelper#number_to_phone.
       #
+      #   number_to_phone("1234567890")         # => "123-456-7890"
+      #
       # Additionally, supports a +:raise+ option that will cause
       # InvalidNumberError to be raised if +number+ is not a valid number:
       #
-      #   number_to_phone("12x34")              # => "12x34"
```
Comment (p8) [inline actionview/lib/action_view/helpers/number_helper.rb:32]: Can you still keeps these, so we don't have to look at `ActiveSupport::NumberHelper` what happens with invalid numbers?

Comment (JuanitoFatas) [reply actionview/lib/action_view/helpers/number_helper.rb:32]: Ah good point, let me restore them 👍🏻

**Review by p8 — APPROVED** [review]

**Comment by p8** [comment]
Thanks @JuanitoFatas !

**Pull request merged**